src/db/addOneRecord.spec.js

```diff
@@ -0,0 +1,43 @@
+import sinon from 'sinon';
+import AWS from 'aws-sdk';
+import { addOneRecord } from './addOneRecord';
+import { config } from '../config/dynamoConfig';
+import * as createFakePK from '../libs/createRestaurantPK';
+import { testParams } from '../constants/testingConstant';
+
+
+describe('db - addOneRecord()', () => {
+    const sandbox = sinon.createSandbox();
+    sandbox.stub(createFakePK, 'createRestaurantPK').returns('fakePK');
+    const promiseStub = sandbox.stub();
+    const clientStub = {
+        put: sandbox.stub().returns({ promise: promiseStub }),
+    };
+
+    beforeEach(() => {
+        sandbox.stub(AWS.DynamoDB, 'DocumentClient').returns(clientStub);
+    });
+
+    afterEach(() => {
+        sandbox.restore();
+    });
+
+    it('Should call DynamoDB PUT method with correct table name', async () => {
+        const name = testParams.name;
+        const phoneNum = testParams.phoneNum;
+        const address = testParams.address;
```

Comment:
use destructuring where possible
```suggestion
        const { name, phoneNum, address } = testParams;
```

Reply:
Thank you :)